src/3d/framegraph/qgspostprocessingrenderview.h

```diff
@@ -0,0 +1,97 @@
+/***************************************************************************
+  qgspostprocessingrenderview.h
+  --------------------------------------
+  Date                 : May 2025
+  Copyright            : (C) 2025 by Benoit De Mezzo and (C) 2020 by Belgacem Nedjima
+  Email                : benoit dot de dot mezzo at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSPOSTPROCESSINGRENDERVIEW_H
+#define QGSPOSTPROCESSINGRENDERVIEW_H
+
+#include "qgsabstractrenderview.h"
+#include "qgspostprocessingentity.h"
+
+namespace Qt3DRender
+{
+  class QTexture2D;
+  class QRenderTargetSelector;
+  class QClearBuffers;
+  class QRenderStateSet;
+  class QRenderCapture;
+} //namespace Qt3DRender
+
+#define SIP_NO_FILE
+
+/**
+ * \ingroup qgis_3d
+ * \brief Container class that holds different objects related to postprocessing rendering
+ *
+ * \note Not available in Python bindings
+ *
+ * The postprocessing buffer render pass is made to copy the postprocessing buffer into
+ * an RGB texture that can be captured into a QImage and sent to the CPU for
+ * calculating real 3D points from mouse coordinates (for zoom, rotation, drag..)
```

Comment:
This is not true as far as I can tell: depth render view is used to capture depth buffer for 3D point calculation, not post-processing.

The main reason for post-processing pass is to apply post-processing effects (shadows, EDL, SSAO).

Render capture is there to capture final rendered image for export - but in my opinion it should not even be in the post-processing render view

Reply:
copied from qgsdepthrenderview ... I forgot to change the desc :(